bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml

```diff
@@ -196,14 +196,12 @@ spec:
                     nullable: true
                     type: integer
                   secret:
-                    description: A Secret with a TLS certificate and key for TLS termination
-                      of the Prometheus endpoint. If the argument is set,
-                      but the Ingress controller is not able to fetch the Secret from
-                      Kubernetes API, the Ingress Controller will fail to start. Format
-                      is namespace/name.
+                    description: Specifies an TLS Secret in the format namespace/name
+                      to use to secure the Prometheus endpoint.
                     type: string
                 required:
                 - enable
+                - secret
```

Comment:
Not sure if this change should be part of the Release PR.

Reply:
I sent a PR https://github.com/nginxinc/nginx-ingress-operator/pull/133

Reply:
It came in when I generated the bundle - I can generate it separately.